builds/e2e/longhaul.yaml

```diff
@@ -349,4 +349,67 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
+
+################################################################################
+  - job: windows_servercore_amd64_moby
+################################################################################
+    displayName: Windows Server Core AMD64 Moby
+    condition: eq(variables['run.windows.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        - Agent.OSName -equals ServerCore
+        - run-long-haul -equals true
+    variables:
+      edgelet.artifact.name: 'iotedged-windows'
+    steps:
+      - checkout: none
+      - task: AzureKeyVault@1
+        displayName: 'Azure Key Vault'
+        inputs:
+          azureSubscription: $(azure.subscription)
+          KeyVaultName: $(azure.keyVault)
+          SecretsFilter: 'edgebuilds-azurecr-io-username,edgebuilds-azurecr-io-pwd,IotHubStressConnString,EventHubStressConnStr,StorageAccountMasterKeyStress,SnitchLongHaulAlertUrl'
+      - task: DownloadBuildArtifacts@0
```

Comment:
Since download artifact tasks can be reused for all x64 windows, please make it as template like templates/longhaul-deploy-windows.yaml for reuse.  AFAIK, you can't move AzureKeyVault task in a template; this will cause VSTS can't resolve the credential during save and will fail when run.

Reply:
Good idea, I'll give this a try!

Reply:
I attempted this and was running into issues with scheduling manual runs. Different builds were being picked up compared to the original config which causes failures to pick up the right assets. This needs more investigation and time.

In addition, this work can be tackled once the IoT Core changes are in as well, so they can all use a template. So for now, I will leave it as is.